py-polars/docs/source/reference/lazyframe/miscellaneous.rst

```diff
@@ -8,6 +8,7 @@ Miscellaneous
 
     LazyFrame.cache
     LazyFrame.collect
+    LazyFrame.sink_parquet
```

Comment:
A bit further down the list. We order them alphabetically.

Reply:
done.